AnnoyancesFilter/sections/cookies_general.txt

```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules that block cookie notifications cookie notifications. Rules should be generic.
```

Comment:
`This section contains the list of the rules...` => `This section contains the list of generic rules...`